orchest-sdk/python/orchest/transfer.py

```diff
@@ -43,19 +63,67 @@ def _check_data_name_validity(name: Optional[str]):
         )
 
 
-class Serialization(Enum):
-    """Possible types of serialization.
+def _interpret_metadata(metadata: str) -> Tuple[str, str, str]:
+    """Interpret and return Orchest SDK metadata.
 
-    Types are:
-        * ``ARROW_TABLE``
-        * ``ARROW_BATCH``
-        * ``PICKLE``
+    Args:
+        metadata: A string that can be interpreted as Orchest SDK
+            metadata. To be considered valid, it must contain the string
+            ``Config.__METADATA__SEPARATOR`` in a way that splitting the
+            string through the separator would result in 3 or 4 elements
+            , in the case of 4 elements, only the last 3 elements are
+            considered. Those three strings must be, in order: a valid
+            string representing a datetime (utc, ISO format), the string
+            representation of a member of the Serialization enum, any
+            string.
+
+    Raises:
+        InvalidMetaDataError: If the input string is invalid.
 
+    Returns:
+        A tuple of 3 strings. The first is the timestamp of when the
+            data related to the metadata was produced (utc, ISO format).
+            The second string is the type of serialization used (See the
+            Serialization enum). The third string is the name with which
+            the data was output.
```

Comment:
This indentation level can be reduced.
```suggestion
        A tuple of 3 strings. The first is the timestamp of when the
        data related to the metadata was produced (utc, ISO format).
        The second string is the type of serialization used (See the
        Serialization enum). The third string is the name with which the
        data was output.
```